### DISCUSSION TITLE
Change prompt only inside of $TERM 'screen'

### DISCUSSION THREAD

**Discussion by spokxer** [ohmyzsh/ohmyzsh#10978] [Help] [answered]
Hi, I am trying to change my prompt inside a $TERM = 'screen' session with an additional prefix, but I am not able to find the correct place where to do it. I am using powerlevel10k as theme.

First I tried to change something in the ~/.oh-my-zsh/plugins/screen/screen.plugin.zsh file, but there only the title can be changed.

I have a two lines prompt with only ">" in the second line and would like to change that to something like "SCREEN>"

What would be your best suggestion to implement this?

**Comment by carlosala** [comment, marked as answer]
probably the easiest thing is to edit the theme itself, adding something like that at the top of your zshrc:
```
[[ $TERM == screen ]] && SCREEN_PREFIX="<whatever you want>
```
And then in the PROMPT variables of the theme adding `$SCREEN_PREFIX` where you want to print that prefix.
With pl10k is actually really difficult to understand where are things happening, but you can still try. I don't know about it, maybe there's some config that you can use, check their repo for confirmation.

**Reply by spokxer** [reply]
really cool suggestion :-) I am one step further

**Reply by spokxer** [reply]
I had to
`nano ~/.p10k.zsh`
and change the following line to
` typeset -g POWERLEVEL9K_MULTILINE_LAST_PROMPT_PREFIX='$SCREEN_PREFIX'` 

Now it works, thanks @carlosala for your help
